settings.py

```diff
@@ -100,3 +100,17 @@
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
 STATIC_URL = '/static/'
+
+LOGIN_REDIRECT_URL = 'website:index'
+LOGOUT_REDIRECT_URL = 'website:index'
+
+AUTH_USER_MODEL = 'website.CustomUser'
+
+
+# E-mail settings
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = os.environ.get('EMAIL_HOST_USER')
```

Comment:
Perhaps use 

```suggestion
EMAIL_HOST_USER = os.environ['EMAIL_HOST_USER']
```

square bracket annotation so that this can fail fast, rather than continue to work if no creds are given (forgotten) so you don't get runtime issues when the time comes to send an email. WDYT?